docs/pages/button.js

```diff
@@ -259,245 +229,481 @@ card(
+`}
+    >
+      <CombinationNew color={['red', 'blue', 'gray', 'transparent']}>
+        {({ color }) => {
+          const map = { red: 'Save', blue: 'Shop', gray: 'Visit', transparent: 'Learn more' };
+          return (
             <Button
-            color={selectedTransparent ? 'transparent' : 'transparentWhiteText'}
-            iconEnd="pin-hide"
-            text="Hide this image"
+              accessibilityLabel={`Example width ${color}`}
+              color={color}
+              text={map[color]}
+              size="lg"
+            />
+          );
+        }}
+      </CombinationNew>
+    </MainSection.Subsection>
+    <MainSection.Subsection
+      columns={2}
+      title="Color on color/image backgrounds"
+      description={`
+  1. White (Primary)
+      High emphasis when placed on color/image backgrounds, used for primary actions in that context.
+  2. Semi-transparent white (Secondary)
+      Medium emphasis when placed on color/image backgrounds, used for secondary actions in that context.
+`}
+    >
+      <MainSection.Card
+        cardSize="md"
+        defaultCode={`
+function WhiteButtonExample() {
+  const [open, setOpen] = React.useState(false);
+  const anchorRef = React.useRef();
+  const viewRef = React.useRef();
+
+  React.useEffect(() => {
+    setOpen(true)
+  });
+
+  return (
+    <ScrollBoundaryContainer>
+      <Box ref={viewRef} width={300} height={220}>
+        <Box display="flex" justifyContent="center" ref={anchorRef}>
+          <Tabs
+            activeTabIndex={1}
+            onChange={() => {}}
+            tabs={[{ href: '#Anchor', text: 'Created'}]}
           />
-          </Box>
-        </Image>
-      </Box>
-      <Box column={4} padding={2}>
-        <Box margin={3}>
-          <Button
-            text="Toggle button color & compare transparency"
-            onClick={() => setSelectedSemiTransparentWhite((currentValue) => !currentValue)}
-            selected={selectedSemiTransparentWhite}/>
         </Box>
-        <Image
-        alt="Image to compare contrast between button color and background."
-          color="rgb(231, 186, 176)"
-          naturalHeight={751}
-          naturalWidth={564}
-          src="https://i.ibb.co/7bQQYkX/stock2.jpg"
-        >
-          <Box padding={2}>
-          <Button
-            color={selectedSemiTransparentWhite ? 'semiTransparentWhite' : 'white'}
-            iconEnd="add-pin"
-            text="Save this image"
-          />
-          </Box>
-          <Box padding={2} position="absolute" bottom right>
-            <Button
-            color={selectedSemiTransparentWhite ? 'white' : 'semiTransparentWhite'}
-            iconEnd="pin-hide"
-            text="Hide this image"
-            inline />
-          </Box>
-        </Image>
       </Box>
-    </Flex>
+      {open &&
```

Comment:
![image](https://user-images.githubusercontent.com/10593890/132906384-1b082363-aa8d-40bc-bd27-4f862e54fdf4.png)
 It took me a while to realize it wasn't a white button but a Tab. Maybe the anchor could be something else.

Reply:
Me thinks we can backlog this as an enhancement - especially given this example was based on one from Popover, so we'd likely want to fix both at once.

Reply:
Agreed with Alberto — see my comment above about an easy fix